src/status_im2/contexts/chat/camera/view.cljs

```diff
@@ -0,0 +1,94 @@
+(ns status-im2.contexts.chat.camera.view
+  (:require
+    [quo2.core :as quo]
+    [quo2.foundations.colors :as colors]
+    [react-native.camera-kit :as camera-kit]
+    [react-native.core :as rn]
+    [react-native.fast-image :as fast-image]
+    [react-native.reanimated :as reanimated]
+    [react-native.safe-area :as safe-area]
+    [reagent.core :as reagent]
+    [utils.i18n :as i18n]
+    [status-im2.contexts.chat.camera.style :as style]
+    [utils.re-frame :as rf]))
+
+(defn- f-zoom-button
+  [{:keys [value current-zoom]}]
+  (let [selected (= @current-zoom value)
+        size     (reanimated/use-shared-value (if selected 37 25))]
+    (rn/use-effect #(reanimated/animate size (if selected 37 25)) [@current-zoom])
+    [rn/touchable-opacity
+     {:on-press #(reset! current-zoom value)
+      :style    style/zoom-button-container}
+     [reanimated/view {:style (style/zoom-button size)}
+      [quo/text
+       {:size   (if selected :paragraph-2 :label)
+        :weight :semi-bold
+        :style  {:color (if selected
+                          colors/system-yellow
+                          colors/white)}}
+       (str value (when selected "x"))]]]))
+
+(defn zoom-button
+  [args]
+  [:f> f-zoom-button args])
+
+(defn snap-button
+  [camera-ref uri]
+  [rn/view {:style style/outer-circle}
+   [rn/touchable-opacity
+    {:on-press (fn [] (camera-kit/capture @camera-ref #(reset! uri %)))
+     :style    style/inner-circle}]])
+
+(defn camera-screen
+  []
+  (let [camera-ref   (atom nil)
+        uri          (reagent/atom nil)
+        current-zoom (reagent/atom "1")]
+    (fn []
+      (let [window                 (rn/get-window)
+            {:keys [width height]} window
+            camera-window-height   (* width 1.33)
+            insets                 (safe-area/get-insets)
+            top                    (/ (- height camera-window-height (:bottom insets)) 2)]
+        [rn/view {:style style/screen-container}
+         [rn/view {:style style/flash-container}
+          [quo/icon :i/flash-camera
+           {:color colors/white
+            :size  24}]]
+         (if @uri
+           [fast-image/fast-image
+            {:style  (style/camera-window width camera-window-height top)
+             :source {:uri @uri}}]
+           [camera-kit/camera
+            {:ref   #(reset! camera-ref %)
+             :style (style/camera-window width camera-window-height top)}])
+         [rn/view {:style (style/zoom-container top insets)}
+          [zoom-button {:value "0.5" :current-zoom current-zoom}]
+          [zoom-button {:value "1" :current-zoom current-zoom}]
+          [zoom-button {:value "2" :current-zoom current-zoom}]
+          [zoom-button {:value "3" :current-zoom current-zoom}]]
+         (if @uri
+           [rn/view {:style (style/confirmation-container insets)}
+            [quo/text
+             {:on-press #(reset! uri nil)
+              :style    {:font-size 17
```

Comment:
Have you considered using `:size :large` instead of overriding `:font-size` style?

Reply:
This seems like something is missing from foundations. Can you align with the desginers? We should not be hardcoding in font-sizes like this afaik.

Reply:
From the message:
https://discord.com/channels/624634427930312714/928625369542713396/1129075900089126973

The designer's idea was to implement the native camera (like the one we used for setting profile photo in Onboarding) for MVP. So, that's why the font size (`17`) and font family (`SF Pro` - the default font of iOS) is creating discrepancies with the typography in Design System.

Should we use the native camera screen for MVP?

Reply:
These designs was supposed to be a native camera, not something that we implement. But, since it is ready we can merge it, and see if the design team would want to have this for now or the native camera. Generally they prefer a custom camera to not break the flow of the app (this one is a custom camera, but not the intended designs :( ). 

Anyways the API implemented can be reused for the new designs.